test/t8_gtest_netcdf_linkage.cxx

```diff
@@ -43,16 +43,12 @@ t8_test_netcdf_linkage ()
   int                 nc_error = nc_create ("FileName", NC_DISKLESS, &ncid);
 
   /* Check for error */
-  SC_CHECK_ABORTF (nc_error == NC_NOERR,
-                   "netcdf error when creating in memory " "file: %s\n",
-                   nc_strerror (nc_error));
+  EXPECT_EQ (nc_error, NC_NOERR);
```

Comment:
You can add a custom failure message using the `<<` operator
See: https://google.github.io/googletest/primer.html#assertions